statistics/betaDistribution.go

```diff
@@ -0,0 +1,36 @@
+package statistics
+
+import (
+	"gonum.org/v1/gonum/stat/distuv"
+)
+
+type BetaDistribution struct {
+	Alpha float64
+	Beta  float64
+}
+
+func (d BetaDistribution) InvCDF(probability float64) float64 {
+	if probability > 1 {
+		panic("nope")
```

Comment:
welp. that is what i get for saying use gamma distribution as the template... it is bad form to panic in a public facing function. We should add a ticket to the project to add a compound type output for the signatures of the continuousdistribution to include an error object so we can properly handle bad inputs and poorly formed distributions